sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,262 @@
+        install_go_1_4
+        export OS_BUILD_ENV_IMAGE="openshift/origin-release:golang-1.4"
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" =~ enterprise-3.[3-4] || "${TARGET_ORIGIN_BRANCH}" =~ release-1.[3-4] ]]; then
+        oct prepare golang --version=1.6
+        export OS_BUILD_ENV_IMAGE="openshift/origin-release:golang-1.6"
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" =~ enterprise-3.5 || "${TARGET_ORIGIN_BRANCH}" =~ release-1.5 ]]; then
+        oct prepare golang --version=1.7.5 -u https://cbs.centos.org/repos/paas7-openshift-origin36-candidate/x86_64/os/
+        export OS_BUILD_ENV_IMAGE="openshift/origin-release:golang-1.7"
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" == "master" ]]; then
+        oct prepare golang --version=1.7.5 -u https://cbs.centos.org/repos/paas7-openshift-origin36-candidate/x86_64/os/
+        export OS_BUILD_ENV_IMAGE="openshift/origin-release:golang-1.7"
+
+      else
+        echo "[FATAL] No Golang version is known for \`${TARGET_ORIGIN_BRANCH}\`.
+              Contact Steve Kuznetsov <skuznets@redhat.com> for assistance."
+        exit 1
+      fi
+  - type: "host_script"
+    title: "install docker"
+    script: |-
+      if [[ "${TARGET_ORIGIN_BRANCH}" == "enterprise-3.1" || "${TARGET_ORIGIN_BRANCH}" == "release-1.1" ]]; then
+        oct prepare docker --version=1.8.2
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" == "enterprise-3.2" || "${TARGET_ORIGIN_BRANCH}" == "release-1.2" ]]; then
+        oct prepare docker --version=1.9.1 --repo=oso-rhui-rhel-server-*
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" == "enterprise-3.2.1" || "${TARGET_ORIGIN_BRANCH}" == "release-1.2.1" ||
+              "${TARGET_ORIGIN_BRANCH}" == "enterprise-3.3" || "${TARGET_ORIGIN_BRANCH}" == "release-1.3" ]]; then
+        oct prepare docker --version=1.10.3 --repo=oso-rhui-rhel-server-*
+          
+      elif [[ "${TARGET_ORIGIN_BRANCH}" =~ enterprise-3.[4-5] || "${TARGET_ORIGIN_BRANCH}" =~ release-1.[4-5] ]]; then
+        oct prepare docker --version=1.12.6 --repo=oso-rhui-rhel-server-* -u https://mirror.openshift.com/enterprise/rhel/dockerextra/x86_64/os/
+
+      elif [[ "${TARGET_ORIGIN_BRANCH}" == "master" ]]; then
+        oct prepare docker --version=1.12.6 --repo=oso-rhui-rhel-server-* -u https://mirror.openshift.com/enterprise/rhel/dockerextra/x86_64/os/
+
+      else
+        echo "[FATAL] No Docker version is known for \`${TARGET_ORIGIN_BRANCH}\`.
+              Contact Steve Kuznetsov <skuznets@redhat.com> for assistance."
+        exit 1
+      fi
+  - type: "host_script"
+    title: "prepare repositories"
+    script: |-
+      oct prepare repositories
+      # Replace origin repo with the specified fork
+      git clone git@github.com:"${ORIGIN_FORK_ID}"/"${ORIGIN_REPO}"
+      oct sync local origin --src "${ORIGIN_REPO}" --dest /data/src/github.com/openshift/origin --branch "${ORIGIN_BRANCH}" --merge-into "${TARGET_ORIGIN_BRANCH}"
```

Comment:
@stevekuznetsov can you vet this change? Doesn't work for me and I can't figure out why
```
failed: [localhost] (item=172.18.1.100) => {
    "changed": true, 
    "cmd": [
        "/usr/bin/git", 
        "push", 
        "ssh://172.18.1.100/data/src/github.com/openshift/origin", 
        "master:master", 
        "--force", 
        "--tags"
    ], 
    "delta": "0:00:00.067998", 
    "end": "2017-05-11 06:46:37.514699", 
    "failed": true, 
    "generated_timestamp": "2017-05-11 06:46:37.534673", 
    "item": "172.18.1.100", 
    "rc": 128, 
    "start": "2017-05-11 06:46:37.446701", 
    "stderr": [
        "Host key verification failed.", 
        "fatal: Could not read from remote repository.", 
        "", 
        "Please make sure you have the correct access rights", 
        "and the repository exists."
    ], 
    "stdout": [], 
    "warnings": []
}
```